src/components/registration/steps.jsx

```diff
@@ -752,7 +759,7 @@ module.exports = {
                                            id: 'registration.validationMaxLength'
                                        })
                                    }}
-                                   required={(this.state.countryChoice === 'us')} />
+                                   required={(this.state.countryChoice === 'us') ? true : 'isFalse'} />
```

Comment:
https://github.com/LLK/scratch-www/blob/develop/src/components/forms/input-hoc.jsx#L14 the "not required" help text is added automatically if the field isn't required. Why is the required state set to `'isFalse'` rather than just `false`?